.circleci/config.yml

```diff
@@ -135,9 +134,10 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: v1-cms-hitech-apd-{{ checksum "web/package-lock.json" }}
-      - restore_cache:
-          key: v1-cms-hitech-api-{{ checksum "api/package-lock.json" }}
+          key: cms-hitech-apd-npm-{{ checksum "web/package-lock.json" }}-{{ checksum "api/package-lock.json" }}
```

Comment:
The deploy steps all run `npm ci`, so restore the npm cache before running them.